components/brave_wallet/browser/json_rpc_service.cc

```diff
@@ -1416,6 +1601,7 @@ void JsonRpcService::GetSupportsInterface(
   auto internal_callback =
       base::BindOnce(&JsonRpcService::OnGetSupportsInterface,
                      weak_ptr_factory_.GetWeakPtr(), std::move(callback));
+  DCHECK(&network_urls_[mojom::CoinType::ETH]);
```

Comment:
I think what you want here is `DCHECK(network_urls_.contains(mojom::CoinType::ETH)`.

Reply:
Updated in baf9235dd6a3fe81b5d71a8143c27d683cde9882